lib/backend/model/rule.go

```diff
@@ -39,16 +39,16 @@ type Rule struct {
 	NotICMPType *int `json:"!icmp_type,omitempty" validate:"omitempty,gte=0,lt=255"`
 	NotICMPCode *int `json:"!icmp_code,omitempty" validate:"omitempty,gte=0,lte=255"`
 
-	SrcTag      string             `json:"src_tag,omitempty" validate:"omitempty,tag"`
-	SrcNet      *net.IPNet         `json:"src_net,omitempty" validate:"omitempty"`
-	SrcNets     []*net.IPNet       `json:"src_nets,omitempty" validate:"omitempty"`
-	SrcSelector string             `json:"src_selector,omitempty" validate:"omitempty,selector"`
-	SrcPorts    []numorstring.Port `json:"src_ports,omitempty" validate:"omitempty,dive"`
-	DstTag      string             `json:"dst_tag,omitempty" validate:"omitempty,tag"`
-	DstSelector string             `json:"dst_selector,omitempty" validate:"omitempty,selector"`
-	DstNet      *net.IPNet         `json:"dst_net,omitempty" validate:"omitempty"`
-	DstNets     []*net.IPNet       `json:"dst_nets,omitempty" validate:"omitempty"`
-	DstPorts    []numorstring.Port `json:"dst_ports,omitempty" validate:"omitempty,dive"`
+	SrcTag        string             `json:"src_tag,omitempty" validate:"omitempty,tag"`
```

Comment:
Was this a `gofmt` change?

Reply:
ya. There is no code change here. i'd added some values, then removed them...should i do 'go fmt' and recommit?

Reply:
yep